docs/types.rst

```diff
@@ -914,6 +921,54 @@ Of course, you can also directly access the members of the struct without
 assigning it to a local variable, as in
 ``campaigns[campaignID].amount = 0``.
 
+
+Types as input and output parameters
+====================================
+
+.. index:: !function parameters, functions, parameters
+
+Allowed function parameter types
+--------------------------------
+
+**All** :ref:`value-types` variables are allowed as input and output function parameter types.
+
+**All** :ref:`reference-types` variables are allowed as input and output function parameter types for internal functions, and for external functions if you enable the experimental ``ABIEncoderV2`` feature.
```

Comment:
except for mappings.

Reply:
@chriseth OK, I made a few further changes to make mappings stand out as their own sort of variable type, as that was throwing me (and maybe others). Let me know if I made correct assumptions or not :)

Nothing internal seems to link to the old heading, and build succeeded.